hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -0,0 +1,234 @@
+    {
+      name: chip_sw_aes_enc
+      desc: '''Verify the AES operation.
+
+            Write a 32-byte key and a 16-byte plain text to the AES registers and trigger the AES
+            computation to start. Wait for the AES operation to complete by polling the status
+            register. Check the digest registers for correctness against the expected digest value.
+            Repeat the test for all three supported key lengths (128/192/256).
+            '''
+      features: [
+        "AES.MODE.ECB",
+      ]
+      stage: V2
+      si_stage: SV1
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: ["chip_sw_aes_enc",
+              "chip_sw_aes_enc_jitter_en"]
+    }
+    {
+      name: chip_sw_aes_multi_block
+      desc: '''Verify the AES operations on multi-block messages.
+            This test should be performed for all supported modes (ECB, CBC, CFB-128, OFB and CTR)
+            and all supported key lengths (128/192/256).
+
+            Note: The number of message blocks can be arbitrarily chosen (at least 2) or randomized
+
+            Note: AES should start this test using the automatic (default) operation. This means
+                  that the MANUAL_OPERATION bit in CTRL_SHADOWED register is set to 0.
+
+            Procedure:
+            -AES should be idle at the begining. Check this by reading the STATUS.IDLE bit.
+            -Set the mode by writing to CTRL_SHADOWED.MODE
+            -Write CTRL_SHADOWED.KEY_LEN to set the key length.
+            -Write to CTRL_SHADOWED.OPERATION to choose encryption or decryption.
+            -Write a key to KEY_SHARE registers
+            -If testing CBC or CTR modes, write the initialization vector to IV registers. Each IV
+             register has to be written at least once. The operation won't start otherwise.
+             The order in which the registers are written does not matter.
+            -Start the AES computation by writing to all DATA_IN registers. The operation
+             (encryption / decryption) starts automatically after every DATA_IN register is written
+             once. The order in which the registers are written does not matter.
+            -Start the operation of the next block by writting to DATA_IN registers and reading
+             the result from the DATA_OUT registers
+                 -The next block operation will not start until all DATA_IN registers are written
+                  and all DATA_OUT registers are read
+                 -Check STATUS.INPUT_READY before writting to DATA_IN registers. These registers
+                  can be written while the operation of the previous block is ongoing.
+                  The order in which the registers are written does not matter.
+                 -Check STATUS.OUTPUT_VALID before reading DATA_OUT
+                  The order in which the registers are read does not matter.
+                 -Repeat this step for all remaining message blocks.
```

Comment:
This is not correct. You just need to write the new data to the DATA_IN registers. The output registers need to be read only to unblock the finishing of the second block (not the start). All this is described in great detail in the AES spec:  https://opentitan.org/book/hw/ip/aes/doc/programmers_guide.html#block-operation and the DIF https://github.com/lowRISC/opentitan-integrated/blob/master/sw/ip/aes/dif/dif_aes.c#L303-L355